### PR TITLE
HHH-17719 Supports boolean as return type of mutation query method

### PR DESCRIPTION
**Pull request by quaff**
https://hibernate.atlassian.net/browse/HHH-17719

### CODE REVIEW

File: documentation/src/main/asciidoc/introduction/Generator.adoc [hunk 1]
```diff
@@ -662,14 +662,20 @@ List<Book> books =
 //                 })
 // ----
 
-An `insert`, `update`, or `delete` query must return `int` or `void`.
+An `insert`, `update`, or `delete` query must return `int` or `boolean` or `void`.
```
Comment (gavinking) [inline documentation/src/main/asciidoc/introduction/Generator.adoc:665]: `int`, `boolean`, or `void`.

**Review by gavinking — CHANGES_REQUESTED** [review]
Looks good!

But please also update the javadoc of the `@HQL` annotation.

Thanks!

File: documentation/src/main/asciidoc/introduction/Generator.adoc (hunk 1 above)
Comment (quaff) [reply documentation/src/main/asciidoc/introduction/Generator.adoc]: @gavinking Done, please review.

**Review by gavinking — APPROVED** [review]
Thanks, LGTM

**Pull request merged**